main.go

```diff
@@ -96,6 +102,26 @@ func updateLicense() {
 	}
 }
 
+func sendTelemetry() {
+	startTime := time.Now()
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		log.Error().Err(err).Msg("Error setting k8s rest config")
+		return
+	}
+
+	clientset, err := kubernetesClient.NewForConfig(config)
+	if err != nil {
+		log.Error().Err(err).Msg("Error creating k8s clientset")
+		return
+	}
+
+	for range time.Tick(time.Minute * 5) {
+		telemetry.Run(startTime, clientset, "kubeshark-hub")
```

Comment:
```suggestion
		telemetry.Run(startTime, K8sWatcher.clientSet, "kubeshark-hub")
```

Pass the `*kubernetes.Clientset` created in here:

https://github.com/kubeshark/hub/blob/d438c094b67e3f1751610e732f6938668e42e5af/pkg/kubernetes/loader.go#L19

Reply:
for this it needs to change struct `K8sWatcher` to have ClientSet exported.

done, I think this won't cause any sideeffects, right?

Reply:
for the record, I'm creating a getter function for clientSet so we dont need to export it